pytorch_lightning/overrides/data_parallel.py

```diff
@@ -160,6 +188,10 @@ def _worker(i, module, input, kwargs, device=None):
                 if not isinstance(input, (list, tuple)):
                     input = (input,)
 
+                if hasattr(module, '_device'):
+                    module._device = device
+                module = module.to(device)
```

Comment:
```suggestion
                module = module.to(device)
```
It is not necessary, since LightningModule already overrides the `.to()` method and sets the device that way